src/components/app/room/index.tsx

```diff
@@ -51,7 +51,7 @@ export class SmoothlyAppRoom {
 						{this.icon ? <smoothly-icon name={this.icon} toolTip={this.label}></smoothly-icon> : this.label}
 					</a>
 				</li>
-				<main ref={e => (this.contentElement = e)}>
+				<main ref={e => (this.contentElement = e)} style={{ overflowY: "auto" }}>
```

Comment:
Is there no easy way to put this styling in css, instead of inline?

Reply:
there must be. but probably not using the > selector. since its moved around in the dom but if marcus doesn't find one i can take a look at it

Reply:
Found a solution.